api/openapi/specs/api.yaml

```diff
@@ -566,6 +568,12 @@ components:
         application/problem+json:
           schema:
             $ref: './common/error_schema.yaml#/components/schemas/Error'
+    NotFound:
```

Comment:
this probably should be in common like here https://github.com/kumahq/kuma/pull/12497/files#diff-b29f834d002dd609516e6d1f777ba0733baaf8e20c70e3aebde1c412c2c3c1d2R72